docs/migration/migrate_7_0.asciidoc

```diff
@@ -23,6 +23,27 @@ Kibana 7.0 will only use the Node.js distribution included in the package.
 
 *Impact:* There is no expected impact unless Kibana is installed in a non-standard way.
 
+[float]
+=== Removed support for using PhantomJS browser for screenshots in Reporting
+*Details:* Since the first release of Kibana Reporting, PhantomJS was used as
+the headless browser to capture screenshots of Kibana dashboards and
+visualizations. In that short time, Chromium has started offering a new
+headless browser library and the PhantomJS maintainers abandoned their project.
+We started planning for a transition in 6.5.0, when we made Chromium the
+default option, but allowed users to continue using Phantom with the
+`xpack.reporting.capture.browser.type: phantom` setting. In 7.0, that setting
+will still exist for compatibility, but the only valid option will be
+`chromium`.
+
+*Impact:* Before upgrading to 7.0, if you have `xpack.reporting.capture.browser.type`
+set in kibana.yml, make sure it is set to `chromium`.
+
+[NOTE]
+============
+Some operating systems can't support Chromium usage. Check for Reporting
+support in your operating system in the Kibana support matrix.
```

Comment:
NOTE: Unlike other stack products, Kibana does not yet have a grid of operating system support in the docs. One should be added: https://github.com/elastic/dev/issues/1048

Reply:
It doesn't sound to me like the support matrix is super straightforward.  I worry that will be delayed (also isn't really tied to a release since it's just docs), but this will remain in the docs and be confusing.  How about just calling out specifically cent os and RHEL 6.x and not mentioning the support matrix? once we do have one we can update this (but if we forget, not the end of the world either, since I'll still be accurate).